tests/migration_test.go

```diff
@@ -432,12 +432,10 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		}, timeout, 1*time.Second).ShouldNot(HaveOccurred())
 
 		By("Waiting until the Migration is Running")
-
 		Eventually(func() bool {
 			migration, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Get(migration.Name, &metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
-			return string(migration.UID) != ""
-
+			return migration.Status.Phase == v1.MigrationRunning
```

Comment:
I'd say it corrects the function :)

Take a look at the "By" description above:
```go
By("Waiting until the Migration is Running")
```

Checking if `string(migration.UID) != ""` does not fulfill this goal. The migration is running if its phase indicates that.

BTW, this is also why I've split `"Immediate migration cancellation"` test into 2 different cases:
- `"Immediate migration cancellation after migration starts running"`
- `"Immediate migration cancellation before migration starts running"`

The second test does not use this function (since it expects the migration to actually start running), but instead creates a migration, waits until its UID is non-empty, then uses `cancelMigration()`.